src/Jackett.Common/Indexers/FileListApi.cs

```diff
@@ -0,0 +1,216 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using AngleSharp.Html.Parser;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig.Bespoke;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Jackett.Common.Utils.Clients;
+using Newtonsoft.Json.Linq;
+using NLog;
+
+namespace Jackett.Common.Indexers
+{
+    public class FileListApi : BaseWebIndexer
+    {
+        public override string[] LegacySiteLinks { get; protected set; } = {
+            "http://filelist.ro/",
+        };
+
+        private string ApiUrl => SiteLink + "api.php";
+
+        private static string Instructions => "Go into your filelist profile and copy the passkey.";
+
+        private new ConfigurationDataFileListApi configData
+        {
+            get => (ConfigurationDataFileListApi)base.configData;
+            set => base.configData = value;
+        }
+
+        public FileListApi(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
+            : base(name: "FileListApi",
+                description: "The best Romanian site.",
+                link: "https://filelist.ro/",
+                caps: TorznabUtil.CreateDefaultTorznabTVCaps(),
```

Comment:
We already add these categories below, so use default initializer and roll in the IMDb Search up here to avoid the compiler error about virtual types in the constructor.
```suggestion
                caps: new TorznabCapabilities {SupportsImdbMovieSearch = true},
```